code/modules/cm_preds/yaut_items.dm

```diff
@@ -923,3 +923,79 @@
 		icon_state = "medicomp_open"
 	else
 		icon_state = "medicomp"
+
+/obj/item/polishing_rag
+	name = "polishing rag"
+	desc = "An astonishingly fine, hand-tailored piece of exotic cloth."
```

Comment:
Give this a hint that it's used for bones, maybe as a pred only text in get_examine_tect